LeezeIt/LeezeItTests/UnitTestPhoneNumberScreen.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UnitTestPhoneNumberScreen.swift
+//  LeezeItTests
+//
+//  Created by Riffat Khan on 9/11/22.
+//
+
+import XCTest
+@testable import LeezeIt
+
+class UnitTestPhoneNumberScreen: XCTestCase {
+    
+    var sut =  PhoneNumberViewController()
+    
```

Comment:
What is sut? can you rename it to something proper

Reply:
Done